api/who/who.proto

```diff
@@ -43,6 +43,18 @@ message GetCaseStatsResponse {
   uint64 ttl = 2;
 }
 
+message StatSnapshot {
+  uint64 epochMsec = 1;
```

Comment:
in the array of StatSnapshots, can I assume that there will be 1 StatSnapshot per 24 hour period (e.g. each epoch is exactly 24 hours apart), or will the client have to aggregate?

Reply:
I think the client should not make that assumption.  The client can control its histogram buckets, so it should do that at any granularity.

Reply:
will that yield accurate counts though? e.g. how does the underlying data source count its daily cases?

Reply:
The concern is: what if the daily counts are recorded at inconsistent times during the day, but intended to record that day's "official" count. Let's say that one day, the daily count is recorded at 10pm and the next day, the daily count is recorded at 9pm. If someone accesses the app in a timezone 3 hours ahead of where the daily counts are recorded, then those 2 StatSnapshots may be aggregated into 1 day, and the previous day would not have any StatSnapshots

Reply:
There's no documentation ofc ;). But it appears the underlying data has exactly 0 or 1 entry for each 24hr period *ending* at midnight GMT.

For example US at 1586304000000 (April 8, 2020 0000hr GMT) has 29,510 deaths which is shown as "April 8, 2020" even though April 8th hasn't ended yet here.

Reply:
Ok, so the client should probably align the days with GMT rather than local time zone?

Reply:
Given we have no API docs, I'd say yes